### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, stat...

### ISSUE THREAD

**Issue by wtgee** [bradleyboy/tidy-tabs-atom#5]
[Enter steps to reproduce below:]

[EDIT: Stripping out actual file names, replaced with <filepath>/<filename>]

Basically, this sort of seems like a problem with atom but it would be nice if tidy tabs handled it. The tab is blank because the file doesn't exist so I want it closed anyway.

1. Create a file in a branch and leave the tab open.
2. Switch to a branch that doesn't have the file.

**Atom Version**: 1.0.7
**System**: Linux Mint
**Thrown From**: [tidy-tabs](https://github.com/bradleyboy/tidy-tabs-atom) package, v0.2.0

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, stat '<filepath>/<filename>'

```
At /usr/share/atom/resources/app.asar/src/pane.js:801

Error: ENOENT: no such file or directory, stat '<filepath>/<filename>'
  at Error (native)
  at fs.statSync (fs.js:846:18)
  at Object.fs.statSync (ATOM_SHELL_ASAR.js:206:16)
  at Object.module.exports.TidyTabs.isCandidate (/home/wtgee/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:69:15)
  at Object.<anonymous> (/home/wtgee/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:87:39)
  at Object.module.exports.TidyTabs.getItemsToClose (/home/wtgee/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:100:10)
  at Object.module.exports.TidyTabs.tidy (/home/wtgee/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:48:5)
  at /home/wtgee/.atom/packages/tidy-tabs/lib/tidy-tabs.coffee:40:18
  at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextBuffer.module.exports.TextBuffer.saveAs (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1025:20)
  at TextBuffer.module.exports.TextBuffer.save (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:992:19)
  at TextEditor.module.exports.TextEditor.save (/usr/share/atom/resources/app.asar/src/text-editor.js:579:26)
  at Pane.module.exports.Pane.saveItem (/usr/share/atom/resources/app.asar/src/pane.js:539:18)
  at Pane.module.exports.Pane.saveActiveItem (/usr/share/atom/resources/app.asar/src/pane.js:522:19)
  at Workspace.module.exports.Workspace.saveActivePaneItem (/usr/share/atom/resources/app.asar/src/workspace.js:609:35)
  at atom-workspace.atom.commands.add.core:save (/usr/share/atom/resources/app.asar/src/workspace-element.js:293:30)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -1:44.8.0 release-notes:show (atom-text-editor.editor.is-focused)
     -0:38.3.0 settings-view:open (atom-text-editor.editor.is-focused)
     -0:31.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:01.1.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ],
    "projectHome": "/home/wtgee/Projects",
    "disabledPackages": [
      "synced-sidebar",
      "vim-mode"
    ]
  },
  "tidy-tabs": {
    "modifiedThreshold": 30,
    "runOnSave": true,
    "accessedThreshold": 15
  }
}
```

### Installed Packages

```coffee
# User
Sublime-Style-Column-Selection, v1.3.0
atom-beautify, v0.28.11
atom-material-syntax, v0.2.3
atom-material-ui, v0.5.21
atomatigit, v1.5.4
autocomplete-paths, v1.0.2
autocomplete-python, v0.5.3
color-picker, v2.0.11
colorful-json, v1.0.0
ctrl-last-tab, v0.2.0
editor-stats, v0.17.0
file-icons, v1.6.6
language-diff, v0.3.1
language-hl7, v0.1.1
language-ipynb, v1.3.0
language-perl-template-toolkit, v0.1.0
markdown-preview-plus, v1.7.0
minimap, v4.12.3
perltidy, v2.2.0
pigments, v0.11.0
project-manager, v1.15.11
python-autopep8, v0.1.3
remote-edit, v1.8.15
smart-tab-name, v1.0.3
tidy-tabs, v0.2.0

# Dev
No dev packages
```

**Comment by bradleyboy** [comment]
Thanks for the report, likely a duplicate of #4. Will investigate.

**Comment by bradleyboy** [comment]
Should be resolved in 0.3.0. Let me know if you still have problems.
